LayoutTests/platform/ios/TestExpectations

```diff
@@ -27,6 +27,7 @@ fast/events/ios [ Pass ]
 fast/events/ios/ipad [ Skip ]
 fast/forms/ios [ Pass ]
 fast/forms/ios/ipad [ Skip ]
+fast/forms/switch/display.html [ Pass ]
```

Comment:
Slightly surprising we don't unskip the entirely directory on iOS?

Reply:
Yeah, I forgot to enable these tests after the iOS implementation landed. I'm also not entirely sure if all are compatible, but let's try and see what fails.

Reply:
Seems like they run just fine.

Reply:
Oh wait. I'm not sure why I thought they'd run fine. Nothing is painted so most of these will fail.

Reply:
I suggest I resolve this by explicitly skipping each test that fails because of painting in the iOS TestExpectations file. Since that's a trivial change, I hope this can be reviewed without that, but let me know if you want me to do that first.

Reply:
Not sure I understand what you mean by "nothing is painted so most of these will fail". Looking at the failing tests on iOS, isn't it just that they need to be rewritten to use `UIHelper`, rather than only using `eventSender`?

In any case, feel free to set up / modify test expectations as necessary in this PR. We can look into rewriting tests and actually getting test parity separately.